src/gui/menu_file.cpp

```diff
@@ -80,8 +80,8 @@ void DissolveWindow::on_FileOpenLocalAction_triggered(bool checked)
         return;
 
     // Request a new file to open
-    QString inputFile =
-        QFileDialog::getOpenFileName(this, "Choose input file to open", QDir().absolutePath(), "Dissolve input files (*.txt)");
+    QString inputFile = QFileDialog::getOpenFileName(this, "Choose input file to open", QDir().absolutePath(),
+                                                     "Dissolve input files (All files (*.*);;Text Files (*.txt))");
```

Comment:
```suggestion
                                                     "Dissolve input files (*.txt);;All Files (*.*)");
```

I would suggest defaulting to the filtered view and letting the users choose an unfiltered one, instead of the other way around.